client/jetpack-cloud/sections/agency-dashboard/downtime-monitoring/contact-editor/hooks.tsx

```diff
@@ -81,3 +85,24 @@ export function useContactFormInputHelpText( type: AllowedMonitorContactTypes )
 		}[ type ];
 	}, [ translate, type ] );
 }
+
+export function useGetSupportedSMSCountries() {
+	// Fetch only once
+	const { data: geoData } = useGeoLocationQuery( {
+		refetchOnWindowFocus: false,
+		staleTime: Infinity,
+	} );
+
+	const countriesList = useSelector( ( state: AppState ) => {
+		const countries = getCountries( state, 'sms' ) ?? [];
+		// Move the user's country to the top of the list
+		const index = countries.findIndex( ( country ) => country.code === geoData?.country_short );
+		if ( index > 0 ) {
+			const country = countries.splice( index, 1 );
+			countries.unshift( country[ 0 ] );
+		}
+		return countries;
+	} );
+
+	return useMemo( () => countriesList, [ countriesList ] );
```

Comment:
I think we don't need to `useMemo` here

Reply:
Fixed it here - https://github.com/Automattic/wp-calypso/pull/78770/commits/0a0aebe5eeb397b35794dfb8c70bbe63bb8741d5